deploy/helm/charts/values.yaml

```diff
@@ -48,7 +48,15 @@ localpv:
   #    cpu: 100m
   #    memory: 128Mi
   nodeSelector: {}
-  tolerations: []
+  tolerations:
+    - effect: NoExecute
+      key: node.kubernetes.io/unreachable
+      operator: Exists
+      tolerationSeconds: 5
+    - effect: NoExecute
+      key: node.kubernetes.io/not-ready
+      operator: Exists
+      tolerationSeconds: 5
```

Comment:
Hmm, what about addition tolerations? If users adds it via set will it remove these? Can we make it in a way that this toleration always stays not matter whatever extra user adds?

Reply:
Right, that would make it easier to upgrade.

Reply:
I've done it a bit differently, somewhat simply. I think this covers all of the cases that the above template function does. What do you think?

Reply:
Looks good to me